mono/mini/interp/transform.c

```diff
@@ -4562,6 +4562,8 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 						}
 					}
 				} else {
+					// Runtime (interp_exec_method_full in interp.c) inserts extra stack to hold return value, before call.
+					simulate_runtime_stack_increase (td, 1);
```

Comment:
Hmm. Shouldn't this be +2 ? As well in the PR from the other day for non recursive VT ?

Reply:
Should it have been +1 before?
I think yes to both.
“This” is .ctor signature or not, I meant to check.

Reply:
I changed them both to 2.